controllers/chat/setSignedChannelAsRead.js

```diff
@@ -15,6 +15,10 @@ const setSignedChannelAsRead = async (req, res) => {
       channelTypeDef = 'group';
       break;
 
+    case CHANNEL_TYPE.TOPIC:
+      channelTypeDef = 'topics';
+      break;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new case for `CHANNEL_TYPE.TOPIC` seems to be correctly implemented. It assigns the string 'topics' to `channelTypeDef` when the channel type is `TOPIC`. However, please ensure that the rest of your code can handle this new case properly and that 'topics' is a valid value for `channelTypeDef` in your application context.
```diff
+    case CHANNEL_TYPE.TOPIC:
+      channelTypeDef = 'topics';
+      break;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->